apps/web-giddh/src/app/shared/eWayBill/create/e-way-bill-create-component.ts

```
@@ -0,0 +1,491 @@
+    * @private
+    * @memberof EWayBillCreateComponent
+    */
+    public ngOnInit(): void {
+        this.initGenerateEwayBillForm();
+        this.initGenerateNewTransporterForm();
+        this.transporterFilterRequest.page = 1;
+        this.transporterFilterRequest.count = 10;
+        this.invoiceService.IsUserLoginEwayBill().pipe(takeUntil(this.destroyed$)).subscribe(res => {
+            if (res?.status === 'success') {
+                this.isUserlogedIn = true;
+                if (res.body && res.body?.gstIn) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)